src/test/java/com/tabnine/integration/StatusBarWidgetTests.java

```diff
@@ -0,0 +1,29 @@
+package com.tabnine.integration;
```

Comment:
this is only health-related status bar tests, so either rename the file or test the entire status bar behavior 😈